docs/learning-powershell/README.md

```diff
@@ -85,7 +85,7 @@ Note that all bash commands should continue working on PowerShell session.
 | Bash                | PowerShell                  | Description
 |:--------------------|:----------------------------|:---------------------
```

Comment:
Can you reformat the table so that the columns line up?  Although markdown rendering will handle it, it doesn't look as nice for someone viewing the file directly from the repo

Reply:
Done!